src/components/Dropdown/index.tsx

```diff
@@ -37,6 +38,7 @@ type TProps = HtmlHTMLAttributes<HTMLInputElement> & {
 
 export const Dropdown = ({
     disabled,
+    dropdownIcon,
```

Comment:
```suggestion
    arrowIcon,
```
can we rename it to this so it will be more clear which icon we mean?

Reply:
or `chevronIcon`